### PR TITLE
added[client]: start chrome from script & use .env in start.sh

### PR DESCRIPTION
**Pull request by 4KevR**
improvement for start.sh

### CODE REVIEW

File: client/.env [hunk 1]
```diff
@@ -1,4 +1,5 @@
-BACKEND_URL=https://localhost
-KICKER_ID=0
+BACKEND_URL=http://localhost
+BACKEND_FRONTEND_URL=http://localhost
```
Comment (TomRomeo) [inline client/.env:2]:
```suggestion
FRONTEND_URL=http://localhost
```

File: client/start.sh [hunk 2]
```diff
@@ -1,5 +1,13 @@
...
+fi
+
+# Start client scripts
 screen -UmdS rfid-scan python3 rfid.py
-screen -UmdS goal-1 python3 goals.py --backend-url "http://localhost" --kid "1"
\ No newline at end of file
+screen -UmdS goal-1 python3 goals.py --backend-url ${BACKEND_URL} --kid ${KICKER_ID}
+
+# Start chrome
+chromium-browser --start-fullscreen ${BACKEND_FRONTEND_URL} &
```
Comment (TomRomeo) [inline client/start.sh:13]:
```suggestion
chromium-browser --start-fullscreen ${FRONTEND_URL} &
```

**Review by TomRomeo — CHANGES_REQUESTED** [review]
pls rename

**Review by TomRomeo — CHANGES_REQUESTED** [review]
Please add a `.gitignore` and exclude the .env file 😊

**Review by TomRomeo — APPROVED** [review]
👍🏼⚡

File: client/start.sh [hunk 3]
```diff
@@ -1,5 +1,13 @@
...
+fi
+
+# Start client scripts
 screen -UmdS rfid-scan python3 rfid.py
-screen -UmdS goal-1 python3 goals.py --backend-url "http://localhost" --kid "1"
\ No newline at end of file
+screen -UmdS goal-1 python3 goals.py --backend-url ${BACKEND_URL} --kid ${KICKER_ID}
+
+# Start chrome
+chromium-browser --start-fullscreen ${FRONTEND_URL} &
```
Comment (darmiel) [inline client/start.sh:13]:
```suggestion
screen -UmdS chrome chromium-browser --start-fullscreen ${FRONTEND_URL}
```

**Review by darmiel — CHANGES_REQUESTED** [review]

**Review by darmiel — APPROVED** [review]

**Pull request merged**